spec/factories/whitehall_export/index_document_factory.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :whitehall_export_index_document, class: Hash do
+    skip_create
+
+    document_id { "123" }
```

Comment:
You could use a sequence for this: https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences that'd make it easier for initialising a few. As you could do build_list(:whitehall_export_index_document, 5) to create 5 rather than having to repeat that 5 times.

Any reason why it's a string?

Reply:
I've updated both the `document_id` (in this factory) and `page_number` (in the `whitehall_export_index` factory) to both use sequence.

It's also now an integer, being a string was an oversight (since I'd wrongly assumed this was the content ID, rather than the Whitehall ID).